dockstore-webservice/src/test/java/io/dockstore/webservice/languages/WDLHandlerTest.java

```diff
@@ -151,6 +152,27 @@ public void testGetToolsForComplexWorkflow() throws IOException {
 
     }
 
+    @Test
+    public void testGetContentWithSyntaxErrors() throws IOException {
+        final WDLHandler wdlHandler = new WDLHandler();
+        final File wdlFile = new File(ResourceHelpers.resourceFilePath("brokenWDL.wdl"));
+        final Set<SourceFile> emptySet = Collections.emptySet();
+
+        // wdlHandler.getContent ultimately invokes toolDAO.findAllByPath from LanguageHandlerEntry.getURLFromEntry for look
+        // up; just have it return null
+        final ToolDAO toolDAO = Mockito.mock(ToolDAO.class);
+        when(toolDAO.findAllByPath(Mockito.anyString(), Mockito.anyBoolean())).thenReturn(null);
+
+        // run test with a WDL descriptor with syntax errors
+        try {
+            wdlHandler.getContent("/brokenWDL.wdl", FileUtils.readFileToString(wdlFile, StandardCharsets.UTF_8), emptySet,
+                LanguageHandlerInterface.Type.TOOLS, toolDAO);
+        } catch (CustomWebApplicationException e) {
+            Assert.assertEquals(e.getResponse().getStatus(), 400);
+            Assert.assertTrue("Should contain parsing error statement, found: " + e.errorMessage, e.errorMessage.contains(WDLHandler.WDL_PARSE_ERROR));
```

Comment:
AssertThat(blah blah).contains(blah blah)

Reply:
Fixed in [WDL](https://github.com/dockstore/dockstore/blob/c5277c33e2a3bf271f3be4e33b742fb646b1a076/dockstore-webservice/src/test/java/io/dockstore/webservice/languages/WDLHandlerTest.java#L174) and [CWL](https://github.com/dockstore/dockstore/blob/c5277c33e2a3bf271f3be4e33b742fb646b1a076/dockstore-webservice/src/test/java/io/dockstore/webservice/languages/CWLHandlerTest.java#L127) test files.